packages/repository/src/repositories/kv.repository.bridge.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import * as legacy from 'loopback-datasource-juggler';
+
+import {Options, DataObject} from '../common-types';
+import {Model} from '../model';
+
+import {KVRepository, AsyncIterator, KVFilter} from './kv.repository';
+
+import {juggler, ensurePromise} from './legacy-juggler-bridge';
+
+/**
+ * An implementation of KVRepository based on legacy loopback-datasource-juggler
+ */
+export class DefaultKVRepository<T extends Model> implements KVRepository<T> {
+  kvModelClass: typeof juggler.KeyValueModel;
+
+  constructor(kvModelClass: typeof juggler.KeyValueModel) {
+    this.kvModelClass = kvModelClass;
```

Comment:
I think this design is not correct. IIRC, in order to create `juggler.KeyValueModel`, one needs a DataSource (or at least a ModelBuilder). That's incompatible with LB4 design, where model classes are standalone and decoupled from repository implementation (legacy juggler bridge or other).

See how DefaultCrudRepository takes a LB4 model definitions and creates a backing LB3 PersistedModel instance:

https://github.com/strongloop/loopback-next/blob/4b14a5ca46ff542002bdbb36c8986f79cc33a5e1/packages/repository/src/repositories/legacy-juggler-bridge.ts#L98

https://github.com/strongloop/loopback-next/blob/4b14a5ca46ff542002bdbb36c8986f79cc33a5e1/packages/repository/src/repositories/legacy-juggler-bridge.ts#L101-L125

IMO, we should do the same for KeyValue models too.